### DISCUSSION TITLE
[Feature request] Public links page

### DISCUSSION THREAD

**Discussion by Arteneko** [go-shiori/shiori#476] [Feature requests]
Hello! It'd be great to have pretty much the same browsable list as a user's logged in list, but as public page, and only with the public links, so we can share a public collection of links.

**Comment by jdaviescoates** [comment]
Yeah, I'd really like this! :100: 

As was using LinkDing which does have a public page of all shared bookmarks, but given there is so much link rot, in many ways I prefer Shiori because of it's ability to create archives. If Shiori had a public links page it'd be the best of both world!

And Shiori can already create shared/ public links, we just need a page where people can browse them all.

Given the data already exists, surely can't be too hard to implement? :shrug:

**Comment by fmartingr** [comment]
The thing is right now what you make public is the archive, not the bookmark itself. And there are requests to not make so easily accesible the public archives already, so the access control may need to change a bit between: internal (logged in only), unlisted (public but not shown on any list) and public (public and shown on a list) or something like that. 

But again, right now the public behavior is for archives, not bookmarks, so that may need to change. Maybe instead of making a list of all public links, a tag could be set as public, creating a unique url for the bookmark list. 

Only ideas in my mind.
